presto-docs/src/main/sphinx/connector/snowflake.rst

```diff
@@ -0,0 +1,111 @@
+connection properties as appropriate for your setup:
+
+.. code-block:: none
+
+    connector.name=snowflake
+    connection-url=jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?db={dbname}
+    connection-user=root
+    connection-password=secret
+
+The ``connection-url`` defines the connection information and parameters to pass
+to the Snowflake JDBC driver.
+
+.. code-block:: text
+
+    connection-url=jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?db={dbname}
+
+The ``connection-user`` and ``connection-password`` are typically required and
+determine the user credentials for the connection, often a service user.
+
+Multiple Snowflake Databases or Accounts
+^^^^^^^^^^^^^^^^^^^^^^
+
+The Snowflake connector can only access a single database within a Snowflake account.
+Thus, if you have multiple Snowflake databases, or want to connect to multiple
+Snowflake accounts, you must configure multiple instances of the Snowflake connector.
+To add another catalog, simply add another properties file to ``etc/catalog``
+with a different name (making sure it ends in ``.properties``). For
+example, if you name the property file ``sales.properties``, Presto
+will create a catalog named ``sales`` using the configured connector.
+
+General Configuration Properties
+---------------------------------
+
+================================================== ==================================================================== ===========
+Property Name                                      Description                                                          Default
+================================================== ==================================================================== ===========
+``user-credential-name``                           Name of the ``extraCredentials`` property whose value is the JDBC
+                                                   driver's user name. See ``extraCredentials`` in `Parameter Reference
+                                                   <https://prestodb.io/docs/current/installation/jdbc.html
+                                                   #parameter-reference>`_.
+
+``password-credential-name``                       Name of the ``extraCredentials`` property whose value is the JDBC
+                                                   driver's user password. See ``extraCredentials`` in `Parameter
+                                                   Reference <https://prestodb.io/docs/current/installation/jdbc.html
+                                                   #parameter-reference>`_.
+
+``case-insensitive-name-matching``                 Match dataset and table names case-insensitively.                    ``false``
+
+``case-insensitive-name-matching.cache-ttl``       Duration for which remote dataset and table names will be
+                                                   cached. Set to ``0ms`` to disable the cache.                         ``1m``
```

Comment:
Is there a standard set of abbreviations for measurements of time in this context? This reads to me as both milliseconds and minutes are acceptable values. I searched but didn't find anything quickly.

Reply:
@steveburnett - Both units milliseconds and minutes are supported.